website/snippets/_v2-sl-prerequisites.md

```diff
@@ -9,7 +9,10 @@
 - Set up the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview) in the integrated tool to import metric definitions. 
   - To access the API and query metrics in downstream tools, you must have a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) account. dbt Core or Developer accounts can define metrics but won't be able to dynamically query them.<br />
 - Understand [MetricFlow's](/docs/build/about-metricflow) key concepts, which powers the latest dbt Semantic Layer. 
-
+- If developing with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation):
+  - Avoid using SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections.
+  - Avoid using Single sign-on (SSO).
```

Comment:
SSO and SSH tunneling are pretty important security features. It may not be a great look for us to say "avoid using" them. Instead, it may be beneficial to state that these features are not currently supported for SL and give a little context as to why (and, if possible, allude to it being supported in the future).

Reply:
Thanks @matthewshaver this isn't ready for docs review so things might change - I'm checking with engineering and product first before the docs review cycle